### PR TITLE
feat(rust): adds cleaned rust service to codebase

### PR DESCRIPTION
**Pull request by dudustri**
- adds rust service
- uses .env.local from the root folder
- prometheus removed

### CODE REVIEW

File: rust-actix/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Added by cargo
```
Comment (Ghmeding) [inline rust-actix/.gitignore:1]: Should we do a global .gitignore instead? I have that in my PR. 
Just so we don't end up with 7 gitignore (1 per project)

File: rust-actix/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,37 @@
+FROM alpine:edge as BUILDER
```
Comment (Ghmeding) [inline rust-actix/Dockerfile:1]: I did not add a Dockerfile to the ExpressJs but just used the Docker compose.

File: rust-actix/Dockerfile_mini-x_test [hunk 3]
```diff
@@ -0,0 +1,19 @@
+FROM python:3.7-alpine
```
Comment (Ghmeding) [inline rust-actix/Dockerfile_mini-x_test:1]: Do we need this Dockerfile for tests? Is it not an overkill?

File: rust-actix/docker-compose-swag.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+version: "3.3"
+services:
+  swag:
```
Comment (Ghmeding) [inline rust-actix/docker-compose-swag.yml:3]: Another name please

File: rust-actix/docker-compose-swag.yml [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    image: lscr.io/linuxserver/swag
+    container_name: swag
+    cap_add:
+      - NET_ADMIN
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - TZ=Europe/London
+      - URL=helge.lol
```
Comment (Ghmeding) [inline rust-actix/docker-compose-swag.yml:12]: Another URL here as well haha

File: rust-actix/src/api/api_structs.rs [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+#[derive(Serialize)]
+pub struct RegisterError {
+    pub status: i32,
+    pub error_msg: String,
+}
+
+#[derive(Serialize)]
+pub struct Message {
```
Comment (Ghmeding) [inline rust-actix/src/api/api_structs.rs:45]: no text or content?

File: rust-actix/src/frontend/static/style.css [hunk 7]
```diff
@@ -0,0 +1,178 @@
+body {
```
Comment (Ghmeding) [inline rust-actix/src/frontend/static/style.css:1]: we need to do the global style.css and do a refactor in all projects for that. The sooner the better

File: rust-actix/swag.conf
Comment (Ghmeding) [inline rust-actix/swag.conf:1]: maybe another name as well?

**Comment by Ghmeding** [comment]
Really good job <3 ! Only general thing is to maybe use another name than 'swag' and to use 'helge.lol' as an URL ;)

File: rust-actix/.gitignore (hunk 1 above)
Comment (dudustri) [reply rust-actix/.gitignore:1]: We can have one gitignore in the root folder + one inside each language. I think it is more organized this way. If we disagree let me know

File: rust-actix/Dockerfile (hunk 2 above)
Comment (dudustri) [reply rust-actix/Dockerfile:1]: You need to build the image some way and to do that you need the docker file.

File: rust-actix/src/api/api_structs.rs (hunk 6 above)
Comment (dudustri) [reply rust-actix/src/api/api_structs.rs:45]: the content is here V - in the frontend they named it text and in the API content but is the same thing cause it is related with the message model (text field); We can change during refactor.

**Review by Ghmeding — APPROVED** [review]

**Pull request merged**